pkg/storage/mto_milestone.go

```diff
@@ -83,8 +83,8 @@ func MTOMilestoneCreate(np sqlutils.NamedPreparer, _ *zap.Logger, MTOMilestone *
 // NOTE: This method is used for updating both Custom and Library-sourced (Common) milestones. It WILL return an error
 // if you try to update one that has both custom info (like a `name`) and a CommonMilestoneKey
 // This is controlled not by application code, but by constraints initially added in this migration: migrations/V185__Add_MTO_Milestone.sql
-func MTOMilestoneUpdate(np sqlutils.NamedPreparer, _ *zap.Logger, MTOMilestone *models.MTOMilestone) (*models.MTOMilestone, error) {
```

Comment:
I think that it's still appropriate to leave this a named preparer (eg either a tx or a store / db). Functionally, this is still fine for the PR, but I don't think we need to force a TX.

Reply:
reverted to np